api.swagger.yaml

```diff
@@ -236,6 +236,30 @@ paths:
           description: An incorrect or malformed token was provided.
         '404':
           description: A file was not found at the supplied S3 URL.
+  /opengraph:
```

Comment:
Could we change this to open_graph, and any other related line items that need correcting?

My bad for spelling it wrong on the original spec...

Reply:
fixed